src/containers/Comments/Comments.js

```diff
@@ -1,34 +1,62 @@
-import React from 'react'
+import React, {  useRef, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { View } from 'react-native'
+import { ScrollView, View } from 'react-native'
 
 import { TextSourceSans } from '../../components/TextSourceSans'
 
 import { Comment } from './Comment'
 
 export const Comments = (props) => {
+  const commentsListRef = useRef()
+  const [commentViewInfo, setCommentViewInfo] = useState([])
   const { t } = useTranslation()
   const { navigation } = props
 
+  // store position of <Comment> and comment id, for later
+  // this happens on mount of this component
+  const handleOnLayout = (index, commentId, y, height) => {
+    commentViewInfo[index] = {commentId, y, height}
+    setCommentViewInfo(commentViewInfo)
+  }
+
+  // after click on 'reply', this "inbetween" handler will be called to
+  // find the comment which has been clicked on. Then the parent's
+  // handleReply will be fired.
+  const handleReplyLayout = (commentContentType, commentObjectPk) => {
+    const parentComment = commentViewInfo
+      .find(cvc => cvc.commentId === commentObjectPk)
+    props.handleReply(
+      commentContentType,
+      commentObjectPk,
+      parentComment.y + parentComment.height + parentComment.height
```

Comment:
not really sure, why the comment height as to be added twice, but on all tested devices, this works.

Reply:
@khamui sorry dude, this doesn't work when testing with a parent comment with a `read more` link expanded, it focuses on the author of the below comment tested on galaxy s10

Reply:
uff yeah, I didn't considered that..will check this case